app/models/manageiq/providers/cloud_manager/orchestration_template_runner.rb

```diff
@@ -36,8 +36,8 @@ def deploy_orchestration_stack
     save!
     my_signal(false, :poll_stack_status, 10)
   rescue StandardError => err
-    _log.log_backtrace(err)
-    my_signal(minimize_indirect, :post_stack_run, err.message, 'error')
+    _log.error("Error deploying orchestration stack : #{err.class} - #{err.message}")
+    my_signal(minimize_indirect, :abort_job, err.message, 'error')
```

Comment:
what is the difference to signal `abort_job`, `error`, vs `finish`?

Reply:
A signal of `abort_job` would indicate that the job did not finish well. It would log an error message, set the job status then call signal `finish`.

Reply:
A signal of `error` just logs an error message then sets the job status, but never calls signal `finish`. 
Seems `abort_job` should be used for all error exits so `dispatch_status` of the job can be put into `finished` that is how we know the job is finished.